AudioBar/MPNowPlayingInfoCenterPatches.swift

```diff
@@ -0,0 +1,35 @@
+import MediaPlayer
+
+extension MPNowPlayingInfoCenter {
```

Comment:
We can split this into two files:

* `MPNowPlayingInfoCenter+ElapsedPlaybackTime.swift`
* `MPNowPlayingInfoCenter+PlaybackDuration.swift`

Once done, we don't need `MARK` comments.